notdiamond/llms/client.py

```diff
@@ -617,21 +617,15 @@ def create(
                     LLMConfig: the best LLM selected by the router
             """
 
-            if model is not None:
```

Comment:
We don't need this code branch? That was my error, if not.

Reply:
Ah, confirming you moved it into the child method.

Reply:
Yea I moved it to the corresponding `invoke` and `ainvoke` methods since `create` and `acreate` are just wrappers for them anyway.